otherlibs/stdlib_alpha/effect.mli

```diff
@@ -156,6 +192,12 @@ module type S = sig
       is passed a [t Handler.t] so that it can perform operations from effect
       [t]. *)
 
+  val portable_fiber :
+    (t Handler.t @ local -> 'a -> 'b) @ portable
+    -> ('a, ('b, unit) Result.Portable.t, unit) Continuation.t @ portable
+  (** [portable_fiber f] works the same as [fiber], but takes a [portable]
+      closure and creates a [portable] continuation. *)
+
```

Comment:
Rather than have `foo`, `portable_foo`, `Portable.foo` and `Portable.portable_foo`, I think it would be better to follow the pattern you used within result: `foo`, `Portable.foo`, `Contended.foo` and `Portable.Contended.foo`. With `Portable` referring to portable continuations and `Contended` referring to functions that where the `Handler.t` can/has crossed a capsule boundary.

I think it would then be better to also have `Portable.Result.t` rather than `Result.Portable.t`.